rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -46,12 +46,13 @@ printf "%s %s\n%s\n\n" "[INFO]" "NVIDIA_DEVICES" "$nvidia_devices"
 {{# jobData.authFile }}
 IFS=$'\r\n' GLOBIGNORE='*' \
   eval 'cred=($(hdfs dfs -cat {{{ jobData.authFile }}}))' \
-  || { echo 'Read authFile Failed' ; exit 1; }
+  || { echo 'Read authFile failed' ; exit 1; }
 docker login --username ${cred[1]} --password ${cred[2]} ${cred[0]} \
-  || { echo 'Authorized Failed' ; exit 1; }
+  || { echo 'Authorized failed' ; exit 1; }
 {{/ jobData.authFile }}
 
-docker pull {{{ jobData.image }}}
+docker pull {{{ jobData.image }}} \
```

Comment:
>docker pull {{{ jobData.image }}} [](start = 0, length = 33)

should we try more than once?

Reply:
Not sure about the retry policy of `docker pull`, it seems that `docker pull` will wait if some errors occur. Does @ydye have any idea?